docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1970,3 +1974,15 @@ String name = "World";
 ----
 
 The preceding code would sets the value of the `name` variable to `appAsecret`.
+
+===  AOT and native image support
+
+Since `4.0.0`, Spring Cloud Config Client supports Spring AOT transformations and GraalVM native images.
+
+WARNING: AOT and native image support is not available for <<config-first-bootstrap, config first bootstrap>> (with `spring.config.use-legacy-processing=true`).
+
+WARNING: Refresh scope is not supported with native images. If you're going to run your config client application as a native image, make sure to set `spring.cloud.refresh.enabled` property to `false`.
+
+WARNING: While building a project that contains Spring Cloud Config Client, it is necessary to make sure that the configuration data source that it connects to (such as, Spring Cloud Config Server, Consul, Zookeeper, Vault, etc.) is available. For example, if you retrieve configuration data from Spring Cloud Config Server, make sure you have its instance running and available at the port indicated in the Config Client setup. This is necessary because the application context is being optimized at build time and requires the target environment to be resolved.
```

Comment:
Change to:

WARNING: While building a project that contains Spring Cloud Config Client, you must make sure that the configuration data source that it connects to (such as, Spring Cloud Config Server, Consul, Zookeeper, Vault, and others) is available. For example, if you retrieve configuration data from Spring Cloud Config Server, make sure you have its instance running and available at the port indicated in the Config Client setup. This is necessary because the application context is being optimized at build time and requires the target environment to be resolved.